logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java

```diff
@@ -0,0 +1,162 @@
+package io.quarkus.ts.logging.jboss;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.zip.ZipException;
+import java.util.zip.ZipFile;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+@Tag("https://issues.redhat.com/browse/QUARKUS-4663")
+public class AccessLoggingIT {
+    private static final String ACCESS_LOG_FILENAME = "appAccess.log";
+    private static final Path ACCESS_LOG_DIR = Path.of("target/AccessLoggingIT/app");
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .withProperties("accessLogging.properties")
+            .withProperty("quarkus.log.handler.file.access-log.path", ACCESS_LOG_FILENAME);
+
+    /**
+     * We want each test to start with no log files from previous tests, so remove them;
+     */
+    @AfterEach
+    public void removeAccessLogs() throws IOException {
+        // App throws exception if we delete log file it is currently working with.
+        // Thus, we need to stop the app while manipulating with log files.
+        app.stop();
+
+        List<Path> accessLogFiles = Files.list(ACCESS_LOG_DIR)
+                .filter(file -> file.getFileName().toString().startsWith(ACCESS_LOG_FILENAME))
+                .toList();
+        for (Path file : accessLogFiles) {
+            Files.delete(file);
+        }
+
+        app.start();
+    }
+
+    @Test
+    public void accessTest() throws IOException {
+        sendRequests(3);
+
+        assertTrue(Files.exists(accessLogPath()), "There should be an access log file.");
+        // check that one request actually match one line in a log file
+        assertEquals(3, Files.lines(accessLogPath()).count(), "There should be 3 lines in access log.");
+
+        assertEquals(1, getLogFileNames().size(), "There should be just one log file");
+    }
+
+    @Test
+    public void logRotationTest() throws IOException {
+        // use zip compression for archived logs
+        setPropertyTo("quarkus.log.handler.file.access-log.rotation.file-suffix", "yyyy-MM-dd.zip");
```

Comment:
We already have this property in `accessLogging.properties`

Given `gzipCompressionTest`, I suggest we drop this option from the properties file

Reply:
Hmm, OK. I removed it from `accessLogging.properties`. It does not break anything and you're right, this way it is cleaner.